contents/docs/cdp/bigquery-export.md

```diff
@@ -6,7 +6,7 @@ tags:
     - bigquery-export
 ---
 
-> This app is in private beta. To join the beta, please [request access](https://app.posthog.com/feature_flags#supportModal=support%3Aapps).
+> This app is currently unavailable while we do [some maintenance](https://github.com/PostHog/posthog/issues/15997). It will be back again soon!
```

Comment:
```suggestion
> This app is currently unavailable while we develop [a new export system](https://github.com/PostHog/posthog/issues/15997). It will be back again soon!
```
I think we are building something new rather than doing maintenance.